pomegranate/FactorGraph.pyx

```diff
@@ -395,7 +395,7 @@ cdef class FactorGraph( GraphModel ):
 			"states": [json.loads(state.to_json()) for state in self.states],
 			"edges": self.edges
 		}
-		return json.dumps(model, separators=separators, indent=indent)
+		return json.dumps(model, separators=separators, indent=indent, sort_keys=True)
```

Comment:
@jmschrei It turns out this was necessary in order for the JSON comparison test of factor graphs (https://github.com/jmschrei/pomegranate/pull/267/files#diff-6ba2f6007ae6a66d1e3941680c09a027R31) to work reliably.